tiles/src/main/java/com/protomaps/basemap/layers/Pois.java

```diff
@@ -134,6 +137,8 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
           kind = sf.getString("natural");
         } else if (sf.hasTag("railway")) {
           kind = sf.getString("railway");
+        } else if (sf.hasTag("highway")) {
```

Comment:
This isn't strictly necessary (since we are just doing highway=bus_stop now), but good to include as a way to future proof later tag value additions.